HomeWork1/task34.py

```diff
@@ -0,0 +1,33 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает,
+# Вам стоит написать программу. Винни-Пух считает, 
+# что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры.
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+
+chant_song=input("Введите кричалку, отделя слова - , а фразы пробелом: ")
+list_vowel='а,я,у,ю,о,е,ё,э,и,ы'
+count=0
+list_chant=list()
+for i in chant_song:
+    if i in list_vowel:
+        count+=1
+    if i==" ":
+        list_chant.append(count)
+        count=0
+if count!=0: list_chant.append(count) 
+if((sum(list_chant)/len(list_chant)==max(list_chant))):
+    print("пара-ра-рам рам-пам-папам па-ра-па-да ")
+else: print("Парам пам-пам ")
+
+print(list_chant)
```

Comment:
Полностью не по PEP8.
```suggestion
chant_song = input("Введите кричалку, отделя слова - , а фразы пробелом: ")
list_vowel = 'а,я,у,ю,о,е,ё,э,и,ы'
count = 0
list_chant = list()
for i in chant_song:
    if i in list_vowel:
        count += 1
    if i == " ":
        list_chant.append(count)
        count = 0
if count != 0:
    list_chant.append(count)
if ((sum(list_chant) / len(list_chant) == max(list_chant))):
    print("пара-ра-рам рам-пам-папам па-ра-па-да ")
else:
    print("Парам пам-пам ")

print(list_chant)
```